src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -330,7 +341,38 @@ def _get_pool_status(
                 });
 
         return (ret, root_map, pool_root)
+    
+    def _update_progress_events(self):
+        osdmap = self.get_osdmap()
+        pools = osdmap.get_pools()
+        for pool_id in list(self._event):
+            ev = self._event[pool_id]
+            pool_data = pools[int(pool_id)]
+            pg_num = pool_data['pg_num']
+            pg_num_target = pool_data['pg_num_target']
+            initial_pg_num = ev._pg_num
+            initial_pg_num_target = ev._pg_num_target
+            progress = (pg_num - initial_pg_num)/(pg_num_target - initial_pg_num)
```

Comment:
dose this need abs() to cover the pg_num lowering case?

Reply:
> does this need abs() to cover the pg_num lowering case?
@jdurgin  I was thinking if pg_num is lowering, it would be < inital_pg_num and pg_num_target would  also be < inital_pg_num

Reply:
Ah you're right, since the new adjustment resets the event the signs should always match